config/routes.rb

```diff
@@ -26,7 +26,11 @@
     end
   end
 
-  resources :checkouts, only: [:index, :create, :show, :update, :destroy]
+  resources :checkouts, only: [:index, :create, :show, :update, :destroy] do
+    collection do
+      delete '', as: '', to: 'checkouts#return_all'
```

Comment:
Is `delete 'return_all'` the same as this longer form?  I'm never sure since routing can be a bit magical.

Reply:
I am pretty sure that when I was trying to implement this `delete 'return_all'` resulted in the path being `checkouts/return_all`. Since we are going to change this to `destroy_all` and implement a different action for our returns, it may be fine to use the shorten it and separate the action from the main `/checkouts` path.